src/acl/HttpReqHeader.h

```diff
@@ -17,7 +17,7 @@ namespace Acl
 {
 
 /// a "req_header" ACL
-class HttpReqHeaderCheck: public ParameterizedNode< ACLData<HttpHeader*> >
+class HttpReqHeaderCheck: public ParameterizedNode< ACLData<const HttpHeader &> >
```

Comment:
Please do not bundle `HttpHeader` type changes into a change documented as being for `HttpReply`. They are very different objects.